data/scripts/eventcallbacks/monster/default_onDropLoot.lua

```diff
@@ -5,6 +5,11 @@ ec.onDropLoot = function(self, corpse)
 		return
 	end
 
+	local percent = 1.0
```

Comment:
I didn't see where this is being used yet

Reply:
I ended up skipping this step, but I believe everything is fine now.